### PR TITLE
Delete AB3 documentation in DG

### PR DESCRIPTION
**Pull request by underthehai**
Update glossary
Fix DeleteSequenceDiagram.png issues
Shift layout in UiClassDiagram

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The sections below give more details of each component.
 
 ### UI component
 
-![Structure of the UI Component](images/UiClassDiagram.png)
+![Structure of the UI Component](images/UIClassDiagram.png)
```
Comment (lyeyixian) [inline docs/DeveloperGuide.md:62]: Hmmm is this image name correct?

Comment (underthehai) [reply docs/DeveloperGuide.md]: okay, fixed!

**Review by jeannetoh99 — APPROVED** [review]
LGTM

**Review by lyeyixian — APPROVED** [review]
LGTM

**Pull request merged**